readthedocs/builds/models.py

```diff
@@ -109,6 +129,15 @@ def __str__(self):
                 pk=self.pk,
             ))
 
+    @property
+    def config(self):
+        last_build = (
```

Comment:
I wonder how long this query will take. Our build table is pretty large, but perhaps the filtering will make it a bit faster. Feels like something we might want to denormalize if we use it a lot.

Reply:
We don't use this, for now, we'll see if we can cache this maybe

Reply:
This method is only used in the tests. Do we need this at all? Did you add it as a helper for debugging or so? Does it worth to keep it here? Where do you think it will be used?

Reply:
This will be used on the api and resolver, it'll be used on the second part of this PR